client/blocks/login/index.jsx

```diff
@@ -3,6 +3,7 @@
  */
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
+import includes from 'lodash/includes';
```

Comment:
It's fine (and preferred) to import from Lodash selectively, i.e.

```js
import { includes } from 'lodash';
```

See: #6539